lectures/oop_intro.md

```diff
@@ -295,7 +295,7 @@ inconsistent?
 The answers are related to the fact that Python aims for consistent style.
 
 In Python, it is common for users to build custom objects --- we discuss how to
-do this [later](python_oop).
+do this {doc}`lecture <python_oop>`.
```

Comment:
@mmcky  Thanks for trying to fix but the sentence no longer makes sense.

Reply:
I'll take care of this.

Reply:
thanks @jstac  -- apologies many of these are `in a later lecture` . Thanks for catching this.